admin-dev/index.php

```diff
@@ -70,6 +71,17 @@
 if (_PS_MODE_DEV_) {
     Debug::enable();
 }
+
+// Loads .env file from the root of project if present
+$dotEnvFile = dirname(__FILE__, 2) . '/.env';
+if (file_exists($dotEnvFile)) {
```

Comment:
.env should already exist

Reply:
Not really!
In fact, this condition should be removed.
Accordingly to DotEnv docs, `.env` can be not found but instead we load `.env.dist`.
As we must gitignore `.env`, we must create a `.env.dist` file in this case.